cheroot/_compat.py

```diff
@@ -14,6 +14,13 @@
 except ImportError:
     import selectors2 as selectors  # noqa: F401  # lgtm [py/unused-import]
 
+try:
+    import backports.socketpair  # noqa: F401  # lgtm [py/unused-import]
+except ImportError:
+    pass
+
+from socket import socketpair  # noqa: F401  # lgtm [py/unused-import]
```

Comment:
I think this version should work even better:
```suggestion
try:
    from backports.socketpair import socket
except ImportError:
    import socket
finally:
    socketpair = socket.socketpair
    del socket
```